app/src/main/res/layout-land/story_chapter_view.xml

```diff
@@ -91,7 +91,7 @@
             android:layout_height="wrap_content"
             android:layout_marginEnd="8dp"
             android:layout_marginBottom="4dp"
-            android:contentDescription="@{@string/chapter_completed}"
+            android:contentDescription="@{String.format(@string/chapter_completed, (viewModel.index + 1), viewModel.name)}"
```

Comment:
Wow, nice find. I didn't realize that we weren't formatting this correctly.

@rt4914 in case something was missed somewhere or if there are other views we might want to check.

Reply:
Actually I had changed this string earlier in my PR https://github.com/oppia/oppia-android/commit/9ad3f85b4648d2bb1ab82c50ca196d0e67e4cf34

But in another it was changed back: https://github.com/oppia/oppia-android/commit/630a433adddc478b8aff20120b7325e168618b44#diff-5e01f7d37a66e4ca03deefc205d8e7008661cdd0284a05aaba1858e6b7bf9103


The final solution is as follows:
1. Do not change `contentDescription` in `story_chapter_view`, i.e., keep this code as `android:contentDescription="@{@string/chapter_completed}"`
2. In `strings.xml` file, change `chapter_completed ` and `chapter_in_progress` values to `Chapter is completed` and `Chapter is in progress` respectively.

This will result in following outcome:

`Chapter 1: What is a Fraction? This is outline/summary of What is a Fraction?` , `Chapter is Completed`, `2 of 3`

<img src="https://user-images.githubusercontent.com/9396084/130128999-18f749c6-2e23-42c8-84c7-d3cbe1c32cbb.png" width="250" />


The current change mentioned in this PR will result in 
`Chapter 1: What is a Fraction? This is outline/summary of What is a Fraction?` , `Chapter 1 with title What is a Fraction is completed`, `2 of 3`
And we can see that there is too much repetition here which does not make sense.


Does this solve the issue?

Reply:
I've reverted all changes related to `chapter_completed_tick` since we can't use distinct content description 
changes requested by @rt4914 would result in having same content description ("Chapter is completed") 

and in this case we can't use the function findObjectByDesc to retrieve the object here.

I've created a separate helper to get the subview from the parent view.
i.e, UiObject2.findObjectByRes()

Reply:
Ack thanks @rt4914. Does this mean R/strings/chapter_completed still needs to be updated to no longer use variables?